share/goa/doc/config.txt

```diff
@@ -139,6 +139,13 @@ Goa evaluates the following configuration variables.
   In this case, a path to a license file can be supplied via the 'license'
   configuration variable, or the '--license' command-line argument.
 
+:debug:
+  If set to 1, Goa uses debug build mode to produce debug build artifacts,
```

Comment:
Suggestions for improving the phrasing of this documentation are gladly welcomed.